doc/developer/design/20210715_table_update.md

```diff
@@ -1,52 +1,158 @@
-# UPDATE and DELETE for tables
+# `UPDATE` and `DELETE` for tables
 
 ## Summary
 
-Support UPDATE and DELETE for tables.
-These cannot be used in transactions and are serialized along with INSERTs.
+Support `UPDATE` and `DELETE` for tables. These cannot be used in transactions
```

Comment:
One of the reasons I formatted this document at one line per sentence is so that if a single sentence changed, the diff wouldn't affect the other things. I'm not sure we have an official policy on this, but this diff is an example of where that'd be useful, because it would more clearly show the changed contents. For example, the longer paragraphs below are completely changed and so require a full eyeball diff to see what has been added. What do you think about going back to one sentence per line to make these updates more clear?

Reply:
Ah, sorry! I have my editor set to auto format everything into this form. Glad to go back and revert this afternoon.